src/ascent/CMakeLists.txt

```diff
@@ -369,8 +369,7 @@ if(MPI_FOUND)
       list(APPEND ascent_sources runtimes/flow_filters/ascent_runtime_babelflow_pmt.cpp)
       list(APPEND ascent_sources runtimes/flow_filters/ascent_runtime_babelflow_comp_utils.hpp)
       list(APPEND ascent_sources runtimes/flow_filters/ascent_runtime_babelflow_comp_utils.cpp)
-      list(APPEND ascent_sources runtimes/flow_filters/ascent_runtime_babelflow_compose.cpp)
-      list(APPEND ascent_sources runtimes/flow_filters/ascent_runtime_babelflow_iso.cpp)
+      list(APPEND ascent_sources runtimes/flow_filters/ascent_runtime_babelflow_compose.cpp) 
```

Comment:
@sshudler just want to double check, are we ready to merge? It seems to only remove one source file from the CMakeLists.txt. (I assume the other changes were present in other another branch?)

Reply:
@cyrush It's strange - it shows 10 commits, but only 1 file changed. Then the last commit in the `bflow-vtkm` branch is "Merge branch 'develop' into bflow-vtkm" by me... I'm not sure - did it happens when I fixed the two conflicts? I don't understand what happend

Reply:
Looks like there is a commit in `develop` called "2020 12 update vtkm (#678)" that includes some (or probably) most of the commits from this branch. I'm not sure how to explain that, but I guess this is why the PR only shows one file has changed

Reply:
ahh, so does that mean we have the bulk of changes in? If so --  are we good to merge this to finish the final changes needed?

Reply:
yes, it appears most of the changes are in and we're good to go with the merge, which will be only this single file

Reply:
great, thanks for clarifying!